src/app/ApplicationTemplate.Shared.Views/Content/Diagnostics/ConfigurationDebuggerView.xaml

```diff
@@ -0,0 +1,103 @@
+﻿<UserControl x:Class="ApplicationTemplate.Views.Content.Diagnostics.ConfigurationDebuggerView"
+			 xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+			 xmlns:local="using:ApplicationTemplate.Views.Content.Diagnostics"
+			 xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+			 xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+			 xmlns:win="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:android="http://uno.ui/android"
+			 xmlns:not_android="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+			 xmlns:xamarin="http://uno.ui/xamarin"
+			 mc:Ignorable="d xamarin android"
+			 d:DesignHeight="300"
+			 d:DesignWidth="400">
+	
+	<UserControl.Resources>
+		
+		<Style x:Key="ConfigurationDebuggerTextBoxStyle"
+			   TargetType="TextBox">
+			<Setter Property="FontSize"
+					Value="11" />
+			<android:Setter Property="FontFamily"
+							Value="monospace" />
+			<not_android:Setter Property="FontFamily"
+								Value="Courier New" />
+			<Setter Property="TextWrapping"
+					Value="Wrap" />
+			<Setter Property="AcceptsReturn"
+					Value="True" />
+			<!-- Workaround for this issue is to not set IsReadOnly to true. https://github.com/unoplatform/uno/issues/9908 -->
+			<!--<Setter Property="IsReadOnly"
```

Comment:
The issue is closed, is it still an issue?

Reply:
Probably not, but I don't want to update Uno in this PR.